src/SlashCommands.tsx

```diff
@@ -239,17 +269,16 @@ export const CommandMap = {
                 const ev = cli.getRoom(roomId).currentState.getStateEvents('m.room.member', cli.getUserId());
                 const content = {
                     ...ev ? ev.getContent() : { membership: 'join' },
-                    displayname: args,
+                    displaycommand: args,
```

Comment:
This looks a bit accidental:
```suggestion
                    displayname: args,
```

Reply:
ah, I was reviewing a specific commit... good catch!